Update devfile and devcontainer webviews to use vscode-elements

tanwigeetika1618 opened the pull request:
Related Jira issue : https://issues.redhat.com/browse/AAP-37624

## Description
Update Devfile and Devcontainer Webviews to Use `@vscode-elements/elements`

## Summary
This PR updates the Devfile and Devcontainer webviews to use `@vscode-elements/elements` instead of `vscode-webview-toolkit`. This change improves consistency across the webviews and aligns with the latest recommendations for VS Code webview UI components.

## Changes
- Replaced `vscode-webview-toolkit` components with `@vscode-elements/elements`

alisonlhart commented on media/contentCreator/createDevcontainerPageStyle.css:
```diff
@@ -29,22 +30,34 @@ h3 {
 
-vscode-text-field {
+vscode-textfield {
     margin-top: 6px;
```
```suggestion
```
Removing this will make the textfield header match the rest of the fields

tanwigeetika1618 replied:
Have adjusted this.

alisonlhart reviewed (commented):
I noticed something when testing that we could adjust: 

On line 136 in createDevcontainerPageApp, the devcontainerPathElement.innerHTML is only changed to the selectedUri, but to match the behavior of the rest of the form it should be 
```
devcontainerPathElement.innerHTML = `${selectedUri}/.devcontainer`;
```

Same for createDevfilePageApp on line 142, should be:
```
devfilePathElement.innerHTML = `${selectedUri}/devfile.yaml`;
```

alisonlhart commented:
These are super minor adjustments. The PR looks great and everything works!!

tanwigeetika1618 commented:
Thanks for catching these, @alisonlhart !  
- Updated `createDevcontainerPageApp` and `createDevfilePageApp` to ensure `devcontainerPathElement` and `devfilePathElement` follow the expected format in `openFolderExplorer` and get appended correctly.  
- Fixed the CSS alignment.  

Let me know if you spot anything else!

tanwigeetika1618 commented on media/contentCreator/createDevfilePageStyle.css:
```diff
@@ -29,16 +30,28 @@ h3 {
 
-vscode-text-area {
+vscode-textarea {
     margin-top: 6px;
```
Have adjusted this.

alisonlhart reviewed (approved):
LGTM!
